src/uu/sync/src/sync.rs

```diff
@@ -88,12 +87,14 @@ mod platform {
             match OpenOptions::new().write(true).open(sliced_name) {
                 Ok(file) => {
                     if FlushFileBuffers(file.as_raw_handle() as HANDLE) == 0 {
-                        crash!(GetLastError() as i32, "failed to flush file buffer");
+                        USimpleError::new(1, format_error_message(GetLastError() as i32,
```

Comment:
please run rustfmt, thanks

Reply:
@sylvestre : Done with new commits. now all tests pass locally(on ubuntu linux) but fail on windows and android. Please help resolve those.

Reply:
found it i need to explicity define a fomat_error_message function... as it is in src/uu/du/src/du.rs.. though not sure why it succeeded on the other linux instances.. adding that function. now.